cmd/cluster/jumphost/init.go

```diff
@@ -0,0 +1,72 @@
+package jumphost
+
+import (
+	"fmt"
+
+	osdctlCommand "github.com/openshift/osdctl/pkg/osdctlCommand"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+// newCmdInit builds a new Jumphost from start to finish
+func newCmdInit(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	ops := &InitOptions{}
+
+	initCmd := &cobra.Command{
+		Use:               "init",
+		Short:             "Initializes a jumphost in a cluster",
+		Args:              cobra.NoArgs,
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(ops.Run())
+		},
+	}
+
+	initCmd.Flags().StringVarP(&ops.keyName, "key-name", "k", "$$USER$$-SRE-Jumphost", "Key Name")
+	return initCmd
+}
+
+type InitOptions struct {
```

Comment:
I'm not terribly sold on keeping the name here `[command]Options`, I'm almost wondering if `[command]Command` might be a better name for it?  So in the case of the key generation command it would be `KeyGenerateCommand`, and would be used within this command as `o.GenerateKeyCommand`, which makes it a bit more clear what it is.  Thoughts?

Reply:
Potential Idea has been thrown into the `GenerateKey` command above.